parmys-plugin/include/odin_types.h

```diff
@@ -0,0 +1,671 @@
+#ifndef ODIN_TYPES_H
```

Comment:
there seems to be a few files copied from VPR project. I don't think this is a good idea to do this. You should rather add a dependency of VPR and use it. See the `systemverilog` plugin (https://github.com/chipsalliance/yosys-f4pga-plugins/blob/main/systemverilog-plugin/Makefile#L35).

Reply:
this one is resolved.